.github/workflows/build-and-quality-checks.yml

```diff
@@ -0,0 +1,27 @@
+name: Code Quality Checks
```

Comment:
please also add the housekeeping action. copy paste from js sdk repo and change branch names

Reply:
I am not in favor of the housekeeping actions.